src/System.Console/src/project.json

```diff
@@ -26,6 +26,27 @@
       "dependencies": {
         "Microsoft.TargetingPack.NETFramework.v4.6": "1.0.1"
       }
+    },
+    "netcore50": {
+      "dependencies": {
+        "Microsoft.TargetingPack.Private.NETNative": "1.0.0-rc4-24126-00",
```

Comment:
I wouldn't expect the Private.NETNative targeting back to be needed here just the WinRT one. Can you please try and remove it and if you cannot please let me know what is needed from it.